pom.xml

```diff
@@ -26,7 +26,7 @@ THE SOFTWARE.
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.424</version>
+    <version>1.596.1</version>
```

Comment:
Not a problem.  I am attempting to address everything you are suggesting.  I would like to get this up and running in an official way.  I did attempt to build with the previous version, but it does error out.  Seeing the errors and some preliminary searches lead me to increase the version to help alleviate the issues, of which it did.  Let's see what other suggest/comment.  Moving up to this version causes deprecation (HudsonTestCase) warnings.  I suspect that all this needs a good cleanup at some point.  I am just jumping in to try and help get functionality that I would really like to have.